packages/theming/atlas/scripts/build.js

```diff
@@ -1,4 +1,7 @@
+const chokidar = require("chokidar");
 const concurrently = require("concurrently");
+const { debounce } = require("lodash");
```

Comment:
What about using the `debounce` from `piw-utils-internal`?

Reply:
As Jordan said, lets avoid lodash. Lots of security issues comes from lodash.

Reply:
I added a debounce function in the script itself. It wasn't easy to reuse `piw-utils-internal` because of the module type. To prevent script compilation I decided to copy it over to the script file.

Reply:
As discussed with Isa, the `piw-utils-internal` package _could_ be changed to output a different module format, allowing this script to import the `debounce` function. This would avoid code duplication. It would need investigation into how it affects code elsewhere, so for now I will approve. We can look into the TS module output later.